logging/src/main/java/life/qbic/logging/impl/logger/LoggerFacade.java

```diff
@@ -0,0 +1,94 @@
+package life.qbic.logging.impl.logger;
+
+import java.util.Objects;
+import life.qbic.logging.api.LogLevel;
+import life.qbic.logging.api.LogMessage;
+import life.qbic.logging.api.Logger;
+import life.qbic.logging.api.Publisher;
+import life.qbic.logging.impl.slf4j.Slf4jWrapper;
+
+/**
+ * Simple implementation of the {@link Logger} interface.
+ * <p>
+ * This class serves as log event hook and splits the logging data flow into two new paths:
+ * <p>
+ * 1. The original logging intention, delegating the logging to an underlying Slf4j API call
+ * <p>
+ * 2. Publishing the log event to a publisher implementation that enables to account for event
+ * specific business actions, for example sending an email to the developer mailing list when an
+ * error is reported.
+ *
+ * The {@link LoggerFacade} does not contain any business logic, other than informing the publisher
+ * and the logging implementation wrapper instance.
+ *
+ * @since 1.0.0
+ */
+public class LoggerFacade implements Logger {
+
+  private final Slf4jWrapper slf4jWrapper;
+
+  private final Publisher publisher;
+
+  public static LoggerFacade from(String name, Publisher publisher) {
+    Objects.requireNonNull(publisher);
+    return new LoggerFacade(name, publisher);
+  }
+
+  public static LoggerFacade from(Class<?> clazz, Publisher publisher) {
+    Objects.requireNonNull(publisher);
+    return from(clazz.getName(), publisher);
+  }
```

Comment:
Why do we need to expose both endpoints? Wouldn't the class one suffice?

Reply:
The `Class` gives you the fully qualified identifier of the class, which sometimes can be undesired. Therefore I provide both possibilities as API and following the convention of the slf4j API.

Reply:
The `Class::getName` provides you with a full identifier of the class. In this case `life.qbic.logging...LoggerFacade`
Is this the desired information?